lib/wallet/bitcoin_tx_detail.dart

```diff
@@ -0,0 +1,52 @@
+import 'package:flutter/material.dart';
+import 'package:intl/intl.dart';
+import 'package:ten_ten_one/bridge_generated/bridge_definitions.dart';
+import 'package:ten_ten_one/utilities/tto_table.dart';
+
+class BitcoinTxDetail extends StatelessWidget {
+  static const subRouteName = 'btc-tx';
+
+  final BitcoinTxHistoryItem transaction;
+
+  const BitcoinTxDetail({required this.transaction, super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    final formatter = NumberFormat();
+    formatter.minimumFractionDigits = 0;
+    formatter.maximumFractionDigits = 0;
+    return Scaffold(
+      appBar: AppBar(title: const Text('Bitcoin Transaction Detail')),
+      body: SafeArea(
+        child: Container(
+            padding: const EdgeInsets.all(20.0),
+            child: TtoTable([
+              TtoRow(
+                  label: "Transaction Id",
+                  value: transaction.txid.substring(0, 7),
```

Comment:
nit: can we do this overflow style instead of a substring: e.g. `c98a39....e261`